src/nodes/chunk_insert_state.h

```diff
@@ -15,6 +15,14 @@
 #include "cache.h"
 #include "cross_module_fn.h"
 
+typedef struct CompressChunkInsertState
+{
+	Relation compress_rel;
+	ResultRelInfo *orig_result_relation_info;
+	CompressSingleRowState *compress_state;
+	bool has_cagg_trigger;
+} CompressChunkInsertState;
```

Comment:
Maybe a few comments would be appropriate, especially considering the fact that this is a public declaration.

Reply:
added.